Felienne/hedy
# Adds ttl for class invitations to automatically delete after 2 weeks

**Description**
In this PR we add a "time to live" value to the class invitations. Currently class invitations live forever, which can be a problem as a user can only have one pending invitation at the time. We add a dedicated `invite_length` value to the config file, setting it at **7 days**. We also perform some minor clean-up on the code by removing some redundant spacing and indentation.

**Fixes**
This PR fixes #2258

**How to test**
Make sure you are a teacher and navigate to any class overview. Invite a new student (that doesn't have a current invitation and is not in your class). Verify that a new "expiration date" column is shown on the invitations overview. 
**The other part we can only verify on Alpha:** 
We have to make sure that the ttl feature of Dynamo works correctly and automatically removes invitations after these 7 days.
